de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/BaseDesktopAppearance.java

```diff
@@ -50,7 +50,7 @@
         @Source("org/iplantc/de/theme/base/client/desktop/header_bg.png")
         ImageResource headerBg();
 
-        @Source("org/iplantc/de/theme/base/client/desktop/mini_logo.png")
```

Comment:
I guess this image is used elsewhere? Otherwise the source file should probably be removed, in any case.